packages/cli/package.json

```diff
@@ -7,8 +7,14 @@
   },
   "bugs": "https://github.com/stoplightio/prism/issues",
   "dependencies": {
+    "@stoplight/http-spec": "^1.0.4",
+    "@stoplight/json": "^3.0.1",
+    "@stoplight/json-ref-resolver": "^2.0.3",
+    "@stoplight/path": "^1.1.0",
     "@stoplight/prism-core": "^3.0.3",
     "@stoplight/prism-http-server": "^3.0.3",
+    "@stoplight/yaml": "^2.8.0",
+    "axios": "^0.19.0",
```

Comment:
Do we need axios or could we get away with node-fetch?

Reply:
We do not need axios specifically, but it's already installed because it's the library being used by the prism http client. I would refrain to add a second http client at the moment.

Reply:
I thought prism switched to fetch 😭 it should, then both should. 

There’s high level http clients and low level http clients. Fetch is low level, and prism should be high level - an alternative to axios. If we can fetch here then switch to fetch in the http client that would be excellent.

Reply:
The client change can be done while we’re refactoring